lib/tasks/import_remediations.rake

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+task import_remediations: :environment do
+  Rails.logger.info('Starting import_remediations job')
+  RemediationsAPI.new(
+    Account.find_by(account_number: ENV['JOBS_ACCOUNT_NUMBER']),
+    ENV['REMEDIATIONS_API']
```

Comment:
Also, IMO we should add path in the RemediationsAPI class to whatever is specified here, so it should be "https://cloud.redhat.com/api/remediations/v1". This will help ensure RemediationsAPI stays flexible to other remediations API paths.

Reply:
I'm removing the variable altogether as we can just use `"#{URI.parse(Settings.host_inventory_url)}#{ENV['PATH_PREFIX']}/remediations/v1/resolutions"`